presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanPrinter.java

```diff
@@ -987,10 +988,11 @@ private void printTableScanInfo(TableScanNode node, int indent)
                     Set<ColumnHandle> outputs = ImmutableSet.copyOf(node.getAssignments().values());
 
                     predicate.getDomains().get()
-                            .entrySet().stream()
-                            .filter(entry -> !outputs.contains(entry.getKey()))
+                            .keySet().stream()
+                            .filter(entry -> !outputs.contains(entry))
+                            .sorted(comparing(entry -> entry.toString()))
                             .forEach(entry -> {
-                                ColumnHandle column = entry.getKey();
+                                ColumnHandle column = entry;
```

Comment:
this variable is no longer needed; let's rename `entry` to `column` and remove this assignment